package.json

```diff
@@ -11,6 +11,7 @@
     "test": "pnpm -r test --filter !transformer-performance-tests",
     "docs": "pnpm -r docs",
     "lint": "pnpm -r lint",
+    "prettier:format": "pnpm -r prettier:format",
```

Comment:
why not just `format`? What else does prettier do?

Reply:
I guess its more explicit this way. Prettier can just output what needs to be changed as opposed to changing it itself. Changing it itself is what prettier:format does.

I guess `format` is an option, but I think I prefer that prettier is mentioned (are there even other formatters? maybe not in which case its pointless)

Reply:
there are others and eventually we could move to one :P some are even currently working on prettier-compatibility. I would prefer we just say "format" and not reveal the implementation detail that we use prettier

Reply:
you can resolve this comment (and merge if all comments are resolved) once you do that